src/networks.json

```diff
@@ -1551,6 +1551,21 @@
     "start": 8821493,
     "logo": "ipfs://QmNkr1UPcBYbvLp7d7Pk4eF3YCsHpaNkfusKZNtykL2EQC"
   },
+  "5000": {
+    "key": "5000",
+    "name": "Mantle",
+    "chainId": 5000,
+    "network": "mainnet",
+    "multicall": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "rpc": [
+      "https://rpc.mantle.xyz"
```

Comment:
Did you tried if this is a full archive node?

Reply:
yes
<img width="1189" alt="Screenshot 2023-08-23 at 12 15 07" src="https://github.com/snapshot-labs/snapshot.js/assets/7693369/d6ad829b-e9f2-485e-bf0e-f69bdfe65615">